Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal class Vehicle : IVehicle
+    {
+        // Manufacturer of the vehicle
+        public string Manufacturer { get; set; }
```

Comment:
The property `model` should be capitalized as `Model` to follow C# naming conventions for properties.